src/components/ContextMenu/Menu.tsx

```diff
@@ -0,0 +1,106 @@
+import React, {
+  PropsWithChildren, useCallback, useEffect, useLayoutEffect, useRef
```

Comment:
`useEffect` doesn't seem to be used anywhere in this file, can be removed.

Reply:
Good catch. We really need to convert this project to use the same structure with linting etc as the others.

Reply:
Agreed! Found a couple of more unused imports, feel free to remove them if you find it necessary.
Or we could wait until proper lint rules are in place.